Makefile

```diff
@@ -1,5 +1,4 @@
 TOOLS_DIR := tools
-PACKAGES_E2E=$(shell go list ./... | grep '/e2etest')
```

Comment:
it's overcomplication: it's possible to use relative test run like `./some_root_dir`